TombEngine/Objects/TR4/Entity/tr4_sas.cpp

```diff
@@ -182,98 +185,80 @@ namespace TEN::Entities::TR4
 
 				if (item->Animation.AnimNumber == Objects[item->ObjectNumber].animIndex + SAS_ANIM_WALK_TO_STAND)
 				{
-					if (abs(AI.angle) >= ANGLE(10.0f))
-					{
-						if (AI.angle >= 0)
-							item->Pose.Orientation.y += ANGLE(10.0f);
-						else
-							item->Pose.Orientation.y -= ANGLE(10.0f);
-					}
-					else
+					if (abs(AI.angle) < ANGLE(10.0f))
 						item->Pose.Orientation.y += AI.angle;
+
+					else if (AI.angle < 0)
+						item->Pose.Orientation.y -= ANGLE(10.0f);
+					else
+						item->Pose.Orientation.y += ANGLE(10.0f);
 				}
-				else if (item->AIBits & MODIFY || Lara.Vehicle != NO_ITEM)
+				else if (item->AIBits == MODIFY || Lara.Vehicle != NO_ITEM)
```

Comment:
Since `AIBits` is a bit field, this check won't work properly if other flags besides `MODIFY` are set.

Reply:
fixed